backend/Tasque.WebAPI/Controllers/EntityController.cs

```diff
@@ -0,0 +1,78 @@
+﻿using AutoMapper;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Tasque.Core.BLL.Services;
+using Tasque.Core.Common.Entities.Abstract;
+
+namespace Tasque.Core.WebAPI.Controllers
+{
+    [ApiController]
+    public abstract class EntityController<TModel, TDto, TService> : ControllerBase
+        where TModel : BaseEntity
+        where TDto : class
+        where TService : IEntityCrudService<TModel>
+    {
+        private readonly TService _service;
+        private readonly Mapper mapper;
+        public EntityController(TService service)
+        {
+            _service = service;
+            var config = new MapperConfiguration(cfg => cfg.CreateMap<TDto, TModel>().ReverseMap());
+            mapper = new Mapper(config);
+        }
+
+        [Route("getOne/{id}")]
```

Comment:
Are you sure about getOne and getAll? Maybe just get and get/{id}?
Also you can omit Route attribute and use [HttpGet(route)]

Reply:
I'm not sure about endpoint naming in general. Why can't it be just single endpoint with different methods?

Reply:
Well, I prefer using naming for endnpoints. It's more intuitive for what's going on (especially during development, when we have to use Postman or other HTTP client) and also it leaves us some space to reuse the same names.

Talking about the first comment, it's definitely possible to do `[HttpGet(route)]`. But `[Route]` was specifically designed for routing, so I prefer to use it.